src/components/PageTable/page-table.scss

```diff
@@ -26,7 +26,7 @@
       td {
         @include carbon--type-style('body-short-01');
         color: $text-01;
-        padding: $spacing-md;
+        padding: $spacing-05;
 
         img[src$='checkmark--glyph.svg'] {
           width: 16px;
```

Comment:
semi related to this PR. Would we want to also update layout tokens? If they're being used or not used, like here?

Reply:
We're not currently using layout tokens anywhere on the site I don't think, but would probably make sense to do that at some point.